### DISCUSSION TITLE
Support changing opening + closing brackets together

### DISCUSSION THREAD

**Discussion by cdce8p** [microsoft/pylance-release#2647] [Enhancement]
It would be nice if `Rename symbol` could support changing opening and closing brackets together. I.e. when refactoring a variable from a tuple to a set or list.
```py
var = (1, 2, 3, 4)

# after 'Rename symbol'
var = {1, 2, 3, 4}
```

--
Alternatively something along the lines of the `HTML mirror cursor` from VS Code: https://code.visualstudio.com/updates/v1_41#_html-mirror-cursor
The user could for example select only the opening bracket. If he/she then replaces it with another, the change could be mirrored to the closing one.

**Comment by jakebailey** [comment]
Thanks for the suggestion.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
